### PR TITLE
Create Help Panel

### PR DESCRIPTION
**Pull request by samuelfangjw**
Change help command to display as a panel instead of a window.

The help panel also displays a command summary.

Screenshot 2021-03-29 at 2 39 43 AM

### CODE REVIEW

File: src/main/java/seedu/address/ui/HelpPanel.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        final ClipboardContent url = new ClipboardContent();
+        url.putString(USERGUIDE_URL);
+        clipboard.setContent(url);
+        mainWindow.setFeedbackToUser(URL_COPIED);
+    }
+
+    private static ObservableList<CommandSyntax> getProjectCommands() {
+        return FXCollections.observableArrayList(
+                new CommandSyntax("addP", "addP n/PROJECT_NAME"),
+                new CommandSyntax("addEto", "addEto PROJECT_INDEX d/DESCRIPTION i/INTERVAL at/DATE"),
```
Comment (vevek) [inline src/main/java/seedu/address/ui/HelpPanel.java:98]: The syntax changed to "addEto PROJECT_INDEX d/DESCRIPTION on/DATE at/TIME w/Y"

**Review by vevek — CHANGES_REQUESTED** [review]
Just the minor Event Syntax

**Review by vevek — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/ui/HelpPanel.java (hunk 1 above)
Comment (samuelfangjw) [reply src/main/java/seedu/address/ui/HelpPanel.java:98]: issue will be resolved in #215, together with other  commands that have been updated recently

**Pull request merged**